pkg/kubectl/cmd/rollingupdate.go

```diff
@@ -81,9 +81,11 @@ func NewCmdRollingUpdate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd := &cobra.Command{
 		Use: "rolling-update OLD_CONTROLLER_NAME ([NEW_CONTROLLER_NAME] --image=NEW_CONTAINER_IMAGE | -f NEW_CONTROLLER_SPEC)",
 		DisableFlagsInUseLine: true,
-		Short:   i18n.T("Perform a rolling update of the given ReplicationController"),
-		Long:    rollingUpdateLong,
-		Example: rollingUpdateExample,
+		Short:      "Perform a rolling update. This command is deprecated, use rollout instead.",
+		Long:       rollingUpdateLong,
+		Example:    rollingUpdateExample,
+		Deprecated: "use rollout instead",
+		Hidden:     true,
```

Comment:
Hmm, not sure why it's still showing up in `kubectl --help`. Maybe it's because we overwrite the printer of the list of available commands?

Reply:
Yeah, I've noticed `run-containers` there as well. Will check it out.